### DISCUSSION TITLE
After I uploaded my document I am receiving this message. You were last verified as a null on Jan.

### DISCUSSION THREAD

**Discussion by dipanadhi** [community/community#103082] [GitHub Education]
### Hi everyone,

Hi, dipanadhi! You were last verified as a null on Jan. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits.

What does this null refers to?

I am trying Student developer pack.

**Comment by NoamS1220** [comment]
I have the same issue

**Comment by tomisouka** [comment]
Did you end up receiving the student developer pack?

**Reply by Bharath-Hegde** [reply]
The users profile says "PRO", so that would mean they ended up receiving it. However, I don't understand the usage of the term "null" here.

**Reply by GCustom** [reply]
null meaning not a student or a teacher, but verified as educational nonetheless.

**Comment by pH-7** [comment]
Correct. GitHub has been displaying this ‘odd’ message for the last several days. The _null_ is likely an internal bug from GitHub, and another value should actually be shown instead. Normally, it states as null until you receive your academic benefits.
However, I understand it can be quite confusing 🫤

**Comment by tent00** [comment]
I have the same issue and I havent recieved the benefits yet

**Reply by Ketanpolawar** [reply]
Hi ,is your issue resolved ? how long did it took for you ?

**Reply by deamichaelis1** [reply]
Same here.  I was really hoping it would be fixed by now.

**Reply by pH-7** [reply]
@Ketanpolawar, @deamichaelis1 From what I've seen, it takes 72 hours in average, and up to 5 days before receiving your academic benefits.

**Reply by deamichaelis1** [reply]
It just went through. Although I think that was 6 days. Or at least on the back end of 5 days for me.

**Reply by Mang30** [reply]
Same problem, I'm currently on my third day

**Reply by tommyn745** [reply]
@Mang30 Same here.

**Reply by tommyn745** [reply]
@Mang30 FYI Just got through. Applied 26 March in the evening just got through today (29)

**Comment by nergisozyol** [comment]
I have the same issue and I havent recieved the benefits yet
